sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -177,6 +179,8 @@
    */
   private boolean enableUncaughtExceptionHandler = true;
 
+  private final @NotNull ExecutorService executorService;
```

Comment:
Left a comment above but ideally we wouldn't have this here

Reply:
answered above